client/state/selectors/get-editor-close-config.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Note: this file was renamed from `client/state/selectors/get-editor-close-url.js` (which shows up as deleted below)